### ISSUE TITLE
A.01.IR06.IR07.supported.crses.as.list

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-interoperable-sds#4]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The Note is misleading because there is no requirement about which CRSs shall be supported by a SDS. The only requirement is to list all the CRSs supported by the service.
'The list of the allowed INSPIRE CRS identifiers is only given as a recommendation, so it's not possible to automatically validate if the CRSes provided fulfill the INSPIRE CRS criteria (ETRS89 based for data inside Europe, ITRS based otherwise).'

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
